src/main/resources/view/MainWindow.fxml

```diff
@@ -13,7 +13,8 @@
 <?import javafx.stage.Stage?>
 
 <?import javafx.scene.layout.HBox?>
-<fx:root minHeight="600" minWidth="450" onCloseRequest="#handleExit" title="VolunTier" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/22" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root minHeight="600" minWidth="750" onCloseRequest="#handleExit" title="VolunTier" type="javafx.stage.Stage"
```

Comment:
I like that you changed the minWidth attribute to prevent users from scaling the application to a size that is unusable as the UI becomes distorted.